content/impressum/contents.lr

```diff
@@ -7,7 +7,7 @@ body:
 #### text-block ####
 text:
 
-## Angaben gemäß § 5 TMG:
+## Angaben gemäß TMG:
```

Comment:
```suggestion
## Angaben gemäß § 5 DDG:
```
Das TMG wurde zum DDG. Entweder wir ändern das komplett oder wir löschen jegliche Referenz auf das TMG, nicht nur den Paragraphen (Angabe des DDG keine Pflicht mehr).

In Zeile 46 wird auch noch 2x das TMG genannt, da muss das auch noch angepasst (TMG --> DDG) werden.

Reply:
Ahh, danke dir! Ich war zu müde :-D. Habe das jetzt in DDG geändert.